src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -118,27 +119,31 @@ public void SynchronizeWithServer_DisconnectedInTheMiddle_ServerInfoIsReusedAndN
         }
 
         [TestMethod]
-        public async Task SynchronizeWithServer_NotInConnectedMode_StoreCleared()
+        public async Task SynchronizeWithServer_NotInConnectedMode_ServerComponentsNotCalledAndStoreCleared()
```

Comment:
Renamed to make it clearer what the test is checking.
It didn't seem worth having a separate test for each server component.